From f6724bfef2515ed5bf66c9a0434655c60a82aae2 Mon Sep 17 00:00:00 2001
From: Thomas Waldmann <tw@waldmann-edv.de>
Date: Fri, 6 Jun 2025 18:35:25 +0200
Subject: [PATCH] msgpack: allow 1.1.1

1.1.1rc1 looked good in testing, so hopefully 1.1.1 will also be ok.
---
 pyproject.toml              | 2 +-
 src/borg/helpers/msgpack.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index f1a3dffb6f..05102a0f0a 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -35,7 +35,7 @@ dependencies = [
     # Please note:
     # using any other msgpack version is not supported by borg development and
     # any feedback related to issues caused by this will be ignored.
-    "msgpack >=1.0.3, <=1.1.0",
+    "msgpack >=1.0.3, <=1.1.1",
     "packaging",
 ]
 
diff --git a/src/borg/helpers/msgpack.py b/src/borg/helpers/msgpack.py
index 5c8cedde16..5c0d1a02b6 100644
--- a/src/borg/helpers/msgpack.py
+++ b/src/borg/helpers/msgpack.py
@@ -137,7 +137,7 @@ def is_slow_msgpack():
 def is_supported_msgpack():
     # DO NOT CHANGE OR REMOVE! See also requirements and comments in pyproject.toml.
     import msgpack
-    return (1, 0, 3) <= msgpack.version <= (1, 1, 0) and \
+    return (1, 0, 3) <= msgpack.version <= (1, 1, 1) and \
            msgpack.version not in []  # < add bad releases here to deny list
 
 
